pkg/clustermanager/cluster_manager.go

```diff
@@ -727,13 +730,30 @@ func (c *ClusterManager) waitForMachineDeploymentReplicasReady(ctx context.Conte
 		timeout = c.machinesMinWait
 	}
 
-	r := retrier.New(timeout)
+	retryPolicy := func(_ int, _ error) (bool, time.Duration) {
+		return true, c.machineBackoff * time.Duration(totalReplicas-readyReplicas)
```

Comment:
This wait time is a little odd. If I have 10 machines and 1 ready it waits 9 seconds. On a second attempt if I have 9 machines ready we still wait 9 seconds? 

There's lots of in-between obviously but this is peculiar, what's your intent?